src/mesh.jl

```diff
@@ -1521,31 +1519,36 @@ function intersection!(a,b,na,nb)
   end
 end
 
-function same_valid_ids(a,b)
-  function is_subset(a,b)
+function find_eq(v::T, b::AbstractVector{T}) where T
```

Comment:
I am not sure about these changes. Why do you need them?

Reply:
You are right. I removed the type declarations. The instability here was mainly due to the operations on the `c` quantity in different scope and also due to nested functions.